cider/tests/plan/CiderPlanValidatorTest.cpp

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include <google/protobuf/util/json_util.h>
+#include <gtest/gtest.h>
+#include <boost/filesystem.hpp>
+#include <string>
+#include "exec/plan/validator/CiderPlanValidator.h"
+#include "util/Logger.h"
+
+namespace bf = boost::filesystem;
+
+std::string getDataFilesPath() {
+  const std::string absolute_path = __FILE__;
+  auto const pos = absolute_path.find_last_of('/');
+  return absolute_path.substr(0, pos) + "/../substrait_plan_files/";
+}
+
+TEST(CiderPlanValidator, InvalidAggAndJoinTest) {
```

Comment:
Should add more test case?

Reply:
will add more in next PR when trying to generate EU and do function lookup.